### DISCUSSION TITLE
load fonts from a folder inside a folder (recurse)

### DISCUSSION THREAD

**Discussion by Obegg** [mpv-player/mpv#14645] [Q&A] [answered]
- Windows 10
- mpv installed in `C:\mpv`
- Folder located on Desktop called "Test"
- Inside "Test" there are 12 files of .mkv which indicates episodes
- Inside "Test" there is a sub-folder called "Subs"
- Inside "Subs" there is a sub-folder called "EP1" 
- Inside "EP1" (and the rest of the sub-folders) there is another sub-folder called "attachments" and in there all the font files are located
- Example: https://animetosho.org/view/ember-rikekoi-science-fell-love-tried-prove-2020.n1469584 (Click on `All Attachments`)

How can mpv load the fonts from "attachments" folder?

**Comment by hooke007** [comment]
dup https://github.com/mpv-player/mpv/issues/9219 https://github.com/mpv-player/mpv/issues/8338

**Reply by Obegg** [reply]
Thank you for your answer, I was searching for issues related to fonts but couldn't find anything relevant.
So I visited the two issues you mentioned and from what I can tell there's currently no way to load fonts the way I mentioned in here, is that right or am I missing something?

**Reply by hooke007** [reply]
Just mod the logic of the folder.

**Reply by Obegg** [reply]
Ah? Dumb it down for me so I can understand.

**Comment by Obegg** [comment, marked as answer]
Ok, so I just moved all fonts a new dedicated folder and I used `sub-fonts-dir` to load them using mpv.
